icxcli/icx/utils.py

```diff
@@ -305,13 +264,25 @@ def check_balance_enough(balance, amount, fee):
 
 
 def check_amount_and_fee_is_valid(amount, fee):
-    if amount <= 0:
+
+    def has_floating_point(str_number):
+        str_number = str(str_number)
+        if '.' in str_number:
+            return True
+        else:
+            return False
+
+    if has_floating_point(amount) or has_floating_point(fee) or not float(amount).is_integer or not float(fee).is_integer:
```

Comment:
```is_integer()``` is right. If you missed the unit test for this function, CI cannot help you. 
http://python-reference.readthedocs.io/en/latest/docs/float/is_integer.html